packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -80,7 +80,12 @@ export const setFieldErrors = assign({
 });
 
 export const setRemoteError = assign({
-  remoteError: (_, event: AuthEvent) => event.data?.message || event.data,
+  remoteError: (_, event: AuthEvent) => {
+    if (event.data.name === 'NoUserPoolError') {
+      return `Configuration error (see console) – please contact the administrator`;
```

Comment:
This can get translated and abstracted, but let's wait until we tackle bigger scope GitHub issue/feature requests.

Reply:
👍